### PR TITLE
Migration

### PR DESCRIPTION
**Pull request by pokornyd**
### Motivation

Migration PR

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [x] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

**Comment by pokornyd** [comment]
tests passing, can't set write permissions to the repo -> codeql failing as a result. resolved by forcing write permissions in the yaml file. remains to update wiki

File: Kontent.Ai.Delivery.Caching.Tests/Fixtures/full_articles.json [hunk 1]
```diff
@@ -144,7 +144,7 @@
         "metadata__twitter_creator": {
           "type": "text",
           "name": "twitter:creator",
-          "value": "@kenticoCloud"
+          "value": "@kontentAi"
```
Comment (IvanKiral) [inline Kontent.Ai.Delivery.Caching.Tests/Fixtures/full_articles.json:147]: not sure, but rather ask not @kontent-ai?

File: LICENSE.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016 Kentico
+Copyright (c) 2022 Kontent.ai s.r.o.
```
Comment (IvanKiral) [inline LICENSE.md:3]: I think that here should be Kontent s.r.o. as it is a legal name of company (also management sdk also has it :D )

File: Kontent.Ai.Delivery.Rx.Tests/Fixtures/articles.json [hunk 3]
```diff
@@ -292,7 +292,7 @@
         "metadata__twitter_creator": {
           "type": "text",
           "name": "twitter:creator",
-          "value": "@kenticoCloud"
+          "value": "@kontentAi"
         },
```
Comment (IvanKiral) [inline Kontent.Ai.Delivery.Rx.Tests/Fixtures/articles.json:296]: probably also kontent-ai?

File: Kontent.Ai.Delivery.Caching.Tests/Fixtures/full_articles.json (hunk 1 above)
Comment (pokornyd) [reply Kontent.Ai.Delivery.Caching.Tests/Fixtures/full_articles.json:147]: I guess if we want it by the book, it should be @Kontent_ai :) https://twitter.com/Kontent_ai will update

File: Kontent.Ai.Delivery.Rx.Tests/Fixtures/articles.json (hunk 3 above)
Comment (pokornyd) [reply Kontent.Ai.Delivery.Rx.Tests/Fixtures/articles.json:296]: dtto

File: LICENSE.md (hunk 2 above)
Comment (pokornyd) [reply LICENSE.md:3]: good catch, will update

**Comment by Simply007** [comment]
Add Readme and Logo same way as for https://github.com/kontent-ai/management-sdk-net/commit/c7cc63d2e92928533306b397aeba484f7d90ab79 (for all csprojs that goes to Nuget)

**Comment by Simply007** [comment]
Talke a look at the CI warnings https://github.com/kontent-ai/delivery-sdk-net/actions/runs/2753457623

**Comment by Simply007** [comment]
Package upgrade will be handled in separate PR (from https://github.com/kontent-ai/delivery-sdk-net/tree/migration)

**Comment by Simply007** [comment]
TODO: rename readmes names to lowercase with dashes => as a best practice (describe somewhere probably /docs/README.md)

**Review by Simply007 — CHANGES_REQUESTED** [review]
See comment above

**Pull request merged**